### ISSUE TITLE
proxy creater

### ISSUE THREAD

**Issue by jjmorales123** [santosderek/Squid-Docker#1]
can i make sneaker/footsite proxies using this project??

**Comment by santosderek** [comment]
Yes you should be able to though I've never done anything related to that. 

You might need to change the squid.conf configuration to suit your needs, but feel free to fork the project and experiment!
